make_prg/from_msa/prg_builder.py

```diff
@@ -103,94 +109,128 @@ def get_sub_alignment_by_list_id(
             sub_alignment = sub_alignment[:, interval[0] : interval[1] + 1]
         return sub_alignment
 
+    @classmethod
```

Comment:
why is this a classmethod?

Reply:
The reason is i)I want this function to be part of the class `PrgBuilder` but ii)I want to be able to test it in isolation, so it doesn't use any `self` attributes/methods. 

If you have an alternative, I'm open!

Reply:
[`@staticmethod`](https://docs.python.org/3/library/functions.html#staticmethod) is probably what you're after. Although this is just minor minor stuff in this context.

Reply:
ah yes. the difference is subtle, staticmethod doesn't receive the `class` object as a function parameter. there may be cases where we want the classmethod to be able to use other methods of the class, so i think it's ok to leave as such. but thanks for pointing out i always forget the two exist